common/windows/delphi/ext/sentry/test/SentryClientVclTest.dpr

```diff
@@ -7,13 +7,13 @@ uses
   sentry in '..\sentry.pas',
   Sentry.Client.Vcl in '..\Sentry.Client.Vcl.pas',
   Keyman.System.KeymanSentryClient in '..\..\..\global\delphi\general\Keyman.System.KeymanSentryClient.pas',
-  KeymanVersion in '..\..\..\global\delphi\general\KeymanVersion.pas',
+  KeymanVersion in '..\..\..\..\..\common\windows\delphi\general\KeymanVersion.pas',
```

Comment:
I don't think `common/common/windows` is correct. Strange that it still compiles...

I think the correct path is `..\..\..\..\..\..\common\windows\delphi\general\KeymanVersion.pas` or `..\..\..\..\..\windows\delphi\general\KeymanVersion.pas` which can be shortened to `..\..\..\general\KeymanVersion.pas`

```suggestion
  KeymanVersion in '..\..\..\general\KeymanVersion.pas',
```